code/game/machinery/syndicatebeacon.dm

```diff
@@ -125,6 +125,9 @@
 	for(var/obj/singularity/singulo in world)
 		if(singulo.z == z)
 			singulo.target = src
+	for(var/obj/singularity/energy_ball/tesla in world)

```

Comment:
You forgot to reset it on deactivation, also these could probably be reduced to in poi_list